README.md

```diff
@@ -1697,6 +1697,8 @@ $ yt-dlp -S "codec:h264"
 # or the best video with best codec if there is no such video
 $ yt-dlp -S "+codec:h264"
 
+# Download the best audio with "main" audio role
+$ yt-dlp -f "ba[role=main]"
```

Comment:
This is unnecessary here IMO, but `role` and its description should be added to the ["Filtering Formats"](https://github.com/yt-dlp/yt-dlp#filtering-formats) section